File: Sorting/Insertion_Sort/insertion_sort.py
def insertion_sort(arr):
	num = len(arr)
	if num > 1:
		for i in range(1, len(arr)):
			key = arr[i]
			j = i-1
			while j >= 0 and arr[j] > key:
				arr[j+1] = arr[j]
				j -= 1
			arr[j+1] = key
	return arr
		
	

File: Sorting/Insertion_Sort/insertion_sort.cpp
#include <iostream>

void insertion_sort(int * arr, int n){
    for (int i = 1; i < n; i++){
        int current = arr[i];
        int j = i - 1;
        while (arr[j] > arr[i] && j >= 0){
            arr[j+1] = arr[j];
            j--;
        }
        arr[j+1] = current;
    }
}